pages/api/[games]/index.js

```diff
@@ -28,6 +29,39 @@ export default async function handler(request, response) {
     } catch (error) {
       return response.status(500).json({ error: error.message });
     }
+  }
+  if (request.method === "POST") {
+    try {
+      const searchTerm = request.body?.searchTerm || "";
+      const { games: offset } = request.query;
```

Comment:
you are using destructuring and classic object value access next to each other. You can unify this to make the code more predictable for readers, and therefore more readable:
```suggestion
      const {searchTerm = ""} = request.body;
      const { games: offset } = request.query;
```